modules/cms/lb.tf

```diff
@@ -124,26 +124,12 @@ resource "aws_security_group" "lb_ingress" {
 
   # For Web traffic to services
   ingress {
-    cidr_blocks = ["0.0.0.0/0"]
-    protocol    = "tcp"
-    from_port   = 80
-    to_port     = 80
-  }
-
-  ingress {
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks =  ["${var.ingress_ips}"]
     protocol    = "tcp"
     from_port   = 443
     to_port     = 443
   }
 
-  ingress {
-    cidr_blocks = ["0.0.0.0/0"]
-    protocol    = "tcp"
-    from_port   = 8800
-    to_port     = 8800
-  }
```

Comment:
this block should be retained, and set to allow traffic from the CMS VPN (otherwise we won't be able to access the circleci replicated console on port 8800)

Reply:
Got it, in that case we should probably separate GIT IPs from VPN IPs, will make that update shortly.